src/extension-runners/chromium.js

```diff
@@ -150,8 +150,28 @@ export class ChromiumExtensionRunner {
       });
     });
 
+    const chromeFlags = [...DEFAULT_CHROME_FLAGS];
+    let startingUrl;
+    var specialStartingUrls = '';
+    if (this.params.startUrl) {
+      const startingUrls = Array.isArray(this.params.startUrl) ?
+        this.params.startUrl : [this.params.startUrl];
+
+      // Remove URLs starting with chrome:// from startingUrls and let bg.js open them instead
+      for (let i = 0; i < startingUrls.length; i++) {
+        if (startingUrls[i].toLowerCase().startsWith('chrome://')) {
+          specialStartingUrls += startingUrls[i] + " "
```

Comment:
The URLs should be appended to an array, not to a string.

Reply:
Thanks for the feedback. I had a hard time with template literals and arrays that's why I used string with spaces, but JSON.stringify() solves both this problem and potential code injection issue you mentioned nicely!